lldb/source/Target/Process.cpp

```diff
@@ -3271,6 +3273,11 @@ Status Process::PrivateResume() {
   if (!GetModID().IsLastResumeForUserExpression())
     ResetExtendedCrashInfoDict();
 
+  if (m_last_run_direction != direction) {
```

Comment:
In the "resolved" discussion we talked about the fact that this will have to change if you ever support a mix of reverse and forward continuation.  Be good to leave a comment to that effect here so we won't have to refind where we need to fix this when the time arrives.

Reply:
Done.